src/ops/math.jl

```diff
@@ -409,3 +409,15 @@ for reduction in [:sum, :prod, :min, :max, :mean]
         Tensor(Operation(desc), 1)
     end
 end
+
+@eval @op function unsorted_segment_sum(data::AbstractTensor,segment_ids,nbuckets; name=nothing)
```

Comment:
the `@eval` here does nothing.

Also for good style place spaces between the arguments, after the commas.